types/src/proptest_types.rs

```diff
@@ -357,7 +357,7 @@ impl SignatureCheckedTransaction {
     ) -> impl Strategy<Value = Self> {
         (keypair_strategy, payload_strategy)
             .prop_flat_map(|(keypair, payload)| {
-                let address = AccountAddress::from_public_key(&keypair.public_key);
+                let address = from_public_key(&keypair.public_key);
```

Comment:
The right way to do these imports is account_address::from_public_key and avoid importing functions directly.